test_runner/regress/test_lfc_prewarm.py

```diff
@@ -42,20 +43,24 @@ def prom_parse(client: EndpointHttpClient) -> dict[str, float]:
 
 
 @pytest.mark.skipif(not USE_LFC, reason="LFC is disabled, skipping")
-@pytest.mark.parametrize("query", QUERY_OPTIONS, ids=["postgres", "compute-ctl"])
-def test_lfc_prewarm(neon_simple_env: NeonEnv, query: LfcQueryMethod):
+@pytest.mark.parametrize("query", QUERY_OPTIONS, ids=QUERY_IDS)
+def test_lfc_prewarm(neon_simple_env: NeonEnv, query: QueryMethod):
```

Comment:
Maybe NIT, maybe not, but to me personally, it's really hard to read this test because of this parametrization. It's easy to read test when it's a sequence of steps. When there are several conditional branching points, it becomes much harder to reason about

Instead, I'd factor out some common setup code, or something

Not gonna push hard on that, though, because `test_lfc_prewarm_under_workload` looks much less branchy although it uses the same approach